# Contains 3 discrete values

# Represents id for the current interaction
int64 id

# Represents human action of either looking at robot or at toy or looking away
int64 human_action 

# Represents human expression (happy, sad, neutral)
int64 human_expression
